docs/SetupDatabaseForAWS.md

```diff
@@ -0,0 +1,25 @@
+# Set up a database for Scalar DB/Scalar DL deployment on AWS
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment on AWS.
+
+## DynamoDB
+
+Amazon DynamoDB is available for use in AWS by default. You do not need to set up anything manually to use it.
+
+### Optional steps
+
+* Configure advanced monitoring services with the [Azure official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/monitoring-automated-manual.html), monitoring is enabled in DynamoDB by default.
+* You should change the `Read/Write Capacity` of DynamoDB tables based on your requirements.
+
+Note:-
+
+* Scalar DB/DL schema loader will configure the same Read/Write Capacity for all tables.
+
+## AWS RDS for MySQL, PostgreSQL, Oracle and SQL Server
+
+In this section, you will create an AWS RDS instance for MySQL/PostgreSQL/Oracle/SQL Server.
```

Comment:
```suggestion
```
If this message is required (, which I don't think so), why no such message for Dynamo?
In the top message, it says, `This guide explains how to set up a database for Scalar DB/Scalar DL deployment on AWS.` so it's good enough.

Reply:
>If this message is required (, which I don't think so), why no such message for Dynamo?

For DynamoDB, a similar sentence was added since it is already available for use and a user doesn't have to set up anything.

>In the top message, it says, This guide explains how to set up a database for Scalar DB/Scalar DL deployment on AWS. so it's good enough.

I will update this section as suggested.
Thank you.

Reply:
I have removed the sentence as per the suggestion.